packages/dashboard/e2e/auth-ready.js

```diff
@@ -15,13 +15,70 @@ async function authReady(timeout = 30000) {
     }, timeout);
     let retryTimer;
     const waitAuthReady = () => {
-      req = http.request('http://localhost:8080/auth/', () => {
+      // initialize network and container related variables.
+      let authIpAddress;
+      const commonNetwork = 'auth_dashboard_network';
+      const commonNetworkSubnet = '172.16.0.0/16';
+      const authContainerIp = '172.16.0.2';
```

Comment:
In case we need it to change this, shouldn't be good for this to be on an `env var`.  Maybe you can set them on this file `packages/dashboard/e2e/env.sh` too and replace

`export REACT_APP_AUTH_CONFIG='{"realm":"master", "clientId":"romi-dashboard", "url":"http://172.16.0.2:8080/auth"}'`

with 

`export REACT_APP_AUTH_CONFIG='{"realm":"master", "clientId":"romi-dashboard", "url":"http://$authContainerIp:8080/auth"}'`

So we will be defining the IP just in one place